fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_flaws.dm

```diff
@@ -130,7 +140,7 @@
 			my_clan = CLAN_VENTRUE
 			to_chat(owner, "<span class='announce'>You have Ranked up enough to learn: You are part of the Ventrue Clan!<br> \
 				* As part of the Ventrue Clan, you are extremely snobby with your meals, and refuse to drink blood from people without a Mind.<br> \
-				* Additionally, you will no longer Rank up. You are now instead able to get a Favorite vassal, by putting a Vassal on the persuasion rack and attempting to Tortute them.<br> \
+				* Additionally, you will no longer Rank up. You are now instead able to get a Favorite vassal, by putting a Vassal on the persuasion rack and attempting to Torture them.<br> \
```

Comment:
im not sure why you capitalize completely random words in every sentence like "Favorite" but whatever

Reply:
lore or something